packages/story-editor/src/components/canvas/karma/keys.karma.js

```diff
@@ -111,10 +106,10 @@ describe('Canvas keys integration', () => {
     );
     await fixture.events.click(colorButton);
     expect(await getSelection()).toEqual([element1.id]);
+    // there are two dialogs, [0] checklist companion and [1]color picker
+    const dialogs = await fixture.screen.findAllByRole('dialog');
+    expect(dialogs.length).toBe(2);
 
-    await waitFor(() => fixture.container.querySelector('[role="dialog"]'), {
-      timeout: 4000,
-    });
```

Comment:
Interestingly I found that the checklist companion dialog was always (at least when I was debugging here) present. So when we would check for role=‘dialog’ we were often getting the checklist dialog and not whatever one we were actually trying to target. ie. color picker.